nucypher/policy/revocation.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, treasure_map, signer: SignatureStamp):
         self.revocations = dict()
         for ursula_address, encrypted_kfrag in treasure_map.destinations.items():
             self.revocations[ursula_address] = RevocationOrder(signer=signer.as_umbral_signer(),
-                                                               ursula_address=ursula_address,
+                                                               staker_address=ursula_address,
```

Comment:
Was this an intentional change?

Reply:
Yes, it's the same `staker_address` as the NodeMetadata has, so I thought it should be named the same too.

Reply:
Hmm - I don't understand.  This change implies the changing of roles from the operater's address to the staking providers address.  At minimum this is a confusing line since `staker` and `ursula` are typically handled as separate actors/addresses.  Are you sure we don't have our wires crossed here?

`staker_address=ursula_address` - huh?

Reply:
Ultimately the revocation API needs a little reinvention so this is not a very important change at this time.

Reply:
The `ursula_address` here is taken from `treasure_map.destinations`, so it is a staker address, not an operator address. That's why I renamed it - makes it more clear.

Reply:
I guess it's worth renaming the loop variable too.

Reply:
Something is certainly incorrect here if we end up with `staking_provider_address=operator_address`, no?

Reply:
There's no operator addresses anywhere here, it's all staking addresses. I guess it the variable just got unfortunately named originally.

Reply:
Ah - Got it! I agree you may as well fixup the loop var.  👍🏻

Reply:
Fixed the loop variable, `nucypher-core` part to be fixed with https://github.com/nucypher/nucypher-core/issues/6